pkg/yamlmeta/parser.go

```diff
@@ -199,7 +199,7 @@ func (p *Parser) assignComments(val interface{}, comments []yaml.Comment, lineCo
 	for _, comment := range comments {
 		comment := &Comment{
 			Data:     comment.Data,
-			Position: p.newPosition(comment.Line, lineCorrection, comment.Data),
+			Position: p.newPosition(comment.Line, lineCorrection, lines[comment.Line]),
```

Comment:
why do we need to be switching to lines. seems like it introduces more possibility that we ll screw something up.